### ISSUE TITLE
upgrading to pandas 0.20.0 breaks geopandas 0.2.1 import

### ISSUE THREAD

**Issue by gboeing** [geopandas/geopandas#443]
I just upgraded pandas to v0.20.0 and it breaks geopandas on import:

```
>>>import geopandas

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-c9ee6bd24dbc> in <module>()
----> 1 import geopandas

C:\Anaconda\lib\site-packages\geopandas\__init__.py in <module>()
----> 1 from geopandas.geoseries import GeoSeries
      2 from geopandas.geodataframe import GeoDataFrame
      3 
      4 from geopandas.io.file import read_file
      5 from geopandas.io.sql import read_postgis

C:\Anaconda\lib\site-packages\geopandas\geoseries.py in <module>()
      6 from pandas import Series, DataFrame
      7 from pandas.core.indexing import _NDFrameIndexer
----> 8 from pandas.util.decorators import cache_readonly
      9 import pyproj
     10 from shapely.geometry import box, shape, Polygon, Point

ImportError: No module named 'pandas.util.decorators'
```

Here's my relevant package info:

```
# packages in environment at C:\Anaconda:
#
geopandas                 0.2.1                    py35_3    conda-forge
pandas                    0.20.0              np112py35_1    conda-forge
```

Confirmed this behavior on Linux and Windows, Python 2.7, 3.5, 3.6.

**Comment by gboeing** [comment]
Looks like pandas made the submodules of pandas.util private earlier today in this commit: https://github.com/pandas-dev/pandas/commit/e99f56e43ab65710a34440067efe372d5f74280b

https://github.com/pandas-dev/pandas/pull/16223

**Comment by jorisvandenbossche** [comment]
Hmm, we should have added a deprecation shim then ... 
Can do a quick 0.2.2 release to fix this.

**Comment by jorisvandenbossche** [comment]
This will be fixed with a pandas 0.20.1. 
But apart from that, we have to fix it here as well of course, so leaving the issue open (but no immediate geopandas 0.2.2 is needed). PR always welcome!

**Comment by ustroetz** [comment]
It is fixed on the pandas side with the release 0.20.01: https://github.com/pandas-dev/pandas/releases/tag/v0.20.1
